hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/overview/overview.tsx

```diff
@@ -115,6 +118,7 @@ export class Overview extends React.Component<Record<string, object>, IOverviewS
         keys: clusterState.keys,
         missingContainersCount,
         openContainers: clusterState.openContainers,
+        deletedKeys: clusterState.deletedKeys,
```

Comment:
change the variable name as these are not deleted keys. These are keys pending for deletion.

Reply:
Done Changes as Suggested by you in new commit